### PR TITLE
Added game loop/guessed double letter

### PR DESCRIPTION
**Pull request by c-werdel**
finished game, added a replay loop and a way to remind the user if they have already guessed that letter. Some minor tweaks were also made like changing user line back into a string to compare with the chosen word.

### CODE REVIEW

File: hangman.py [hunk 1]
```diff
@@ -19,15 +19,20 @@ def letter_checker(word_choice):
...
     # word_choice = list(word_choice) 
     user_line = list(word_completion)
-    wrong_guesses = []
-    print("Your word is", user_line)
-    playing = True
-
-    while playing:
-        print("Letters guessed wrong: " , wrong_guesses,) 
+    guessed = []
+    print(word_choice)
```
Comment (dev-gilbride) [inline hangman.py:23]: This line prints the word before you play, pretty sure it was for testing. You should remove it.

File: hangman.py [hunk 2]
```diff
@@ -19,15 +19,20 @@ def letter_checker(word_choice):
     word_completion = "_" *len(word_choice) 
     # word_choice = list(word_choice)
```
Comment (dev-gilbride) [inline hangman.py:20]: Remove commented line.

File: hangman.py [hunk 3]
```diff
@@ -36,34 +41,39 @@ def letter_checker(word_choice):
...
-
-        if user_line == word_choice: #if user line matches word_choice
+    
+        if "".join(user_line) == word_choice: #if user line matches word_choice
             print("correct you guessed", user_line, "and the word was", word_choice,)
-
-        if fail_count == fail_limit:
-            print("Sorry you lost the word was" , word_choice)
+            break
+        #put a list back into a string
```
Comment (dev-gilbride) [inline hangman.py:57]: What is this referencing? It doesn't relate to the code around it. Perhaps line 54?

File: hangman.py [hunk 4]
```diff
@@ -36,34 +41,39 @@ def letter_checker(word_choice):
...
+        if fail_count >= fail_limit:
+            print("Sorry you lost the word was" , word_choice)
+            break
 
-    # if new_game[0].lower()=='y':
-    #     playing = True
-    #     continue
-    # else:
-    #     print("Thanks for playing")
-    #     break
```
Comment (dev-gilbride) [inline hangman.py:62]: Remove commented lines.

File: hangman.py [hunk 5]
```diff
@@ -19,15 +19,20 @@ def letter_checker(word_choice):
...
+    print(word_choice)
+        
+    while True:
+        print("Your word is", user_line)
+        print("Letters guessed wrong: " , guessed,) 
         print("You have used up", + fail_count, 'guess of', fail_limit, 'total guesses') 
         guess = input("Guess a letter ").upper()
         
+        if guess in guessed or guess in user_line:
+            print("you already guessed", guess)
```
Comment (dev-gilbride) [inline hangman.py:32]: Capitalize to "You already..."

**Review by dev-gilbride — CHANGES_REQUESTED** [review]

**Comment by c-werdel** [comment]
corrected making new push

**Pull request closed without merging**